frontend_tests/node_tests/recent_senders.js

```diff
@@ -23,8 +23,8 @@ function make_stream_message({stream_id, topic, sender_id}) {
     return message;
 }
 
-const message_list = mock_esm("../../static/js/message_list", {
-    all: {
+const {all_messages_data} = mock_esm("../../static/js/all_messages_data", {
```

Comment:
#17879 will eliminate the need for any mock here.  (just a heads up if it gets merged first)